mappings/net/minecraft/client/render/GameRenderer.mapping

```diff
@@ -3,6 +3,60 @@ CLASS net/minecraft/class_757 net/minecraft/client/render/GameRenderer
+	FIELD field_29364 renderTypeCutoutMippedShader Lnet/minecraft/class_5944;
+	FIELD field_29365 renderTypeCutoutShader Lnet/minecraft/class_5944;
+	FIELD field_29366 renderTypeTranslucentShader Lnet/minecraft/class_5944;
+	FIELD field_29367 renderTypeEntityGlintShader Lnet/minecraft/class_5944;
+	FIELD field_29368 renderTypeEntityGlintDirectShader Lnet/minecraft/class_5944;
+	FIELD field_29369 renderTypeTextShader Lnet/minecraft/class_5944;
+	FIELD field_29370 renderTypeTextSeeThroughShader Lnet/minecraft/class_5944;
+	FIELD field_29371 renderTypeLightningShader Lnet/minecraft/class_5944;
+	FIELD field_29372 renderTypeTripwireShader Lnet/minecraft/class_5944;
+	FIELD field_29373 renderTypeEndPortalShader Lnet/minecraft/class_5944;
+	FIELD field_29374 renderTypeEndGatewayShader Lnet/minecraft/class_5944;
+	FIELD field_29375 renderTypeLinesShader Lnet/minecraft/class_5944;
+	FIELD field_29376 renderTypeCrumblingShader Lnet/minecraft/class_5944;
+	FIELD field_29377 renderTypeTranslucentMovingBlockShader Lnet/minecraft/class_5944;
+	FIELD field_29378 renderTypeTranslucentNoCrumblingShader Lnet/minecraft/class_5944;
+	FIELD field_29379 renderTypeArmorCutoutNoCullShader Lnet/minecraft/class_5944;
+	FIELD field_29380 renderTypeEntitySolidShader Lnet/minecraft/class_5944;
+	FIELD field_29381 renderTypeEntityCutoutShader Lnet/minecraft/class_5944;
+	FIELD field_29382 renderTypeEntityCutoutNoNullShader Lnet/minecraft/class_5944;
+	FIELD field_29383 renderTypeEntityCutoutNoNullZOffsetShader Lnet/minecraft/class_5944;
+	FIELD field_29384 renderTypeItemEntityTranslucentCullShader Lnet/minecraft/class_5944;
+	FIELD field_29385 renderTypeEntityTranslucentCullShader Lnet/minecraft/class_5944;
+	FIELD field_29386 renderTypeEntityTranslucentShader Lnet/minecraft/class_5944;
+	FIELD field_29387 renderTypeEntitySmoothCutoutShader Lnet/minecraft/class_5944;
+	FIELD field_29388 renderTypeBeaconBeamShader Lnet/minecraft/class_5944;
+	FIELD field_29389 renderTypeEntityDecalShader Lnet/minecraft/class_5944;
+	FIELD field_29390 renderTypeEntityNoOutlineShader Lnet/minecraft/class_5944;
+	FIELD field_29391 renderTypeEntityShadowShader Lnet/minecraft/class_5944;
+	FIELD field_29392 renderTypeEntityAlphaShader Lnet/minecraft/class_5944;
+	FIELD field_29393 renderTypeEyesShader Lnet/minecraft/class_5944;
+	FIELD field_29394 renderTypeEnergySwirlShader Lnet/minecraft/class_5944;
+	FIELD field_29395 renderTypeLeashShader Lnet/minecraft/class_5944;
+	FIELD field_29396 renderTypeWaterMaskShader Lnet/minecraft/class_5944;
+	FIELD field_29397 renderTypeOutlineShader Lnet/minecraft/class_5944;
+	FIELD field_29398 renderTypeArmorGlintShader Lnet/minecraft/class_5944;
+	FIELD field_29399 renderTypeArmorEntityGlintShader Lnet/minecraft/class_5944;
+	FIELD field_29400 renderTypeGlintTranslucentShader Lnet/minecraft/class_5944;
+	FIELD field_29401 renderTypeGlintShader Lnet/minecraft/class_5944;
+	FIELD field_29402 renderTypeGlintDirectShader Lnet/minecraft/class_5944;
```

Comment:
Should we use `renderType` prefix or just dump them, or suffix them like `xxLayerShader`?